src/main/jbake/content/hawkular-apm/index.adoc

```diff
@@ -0,0 +1,124 @@
+|Javascript |https://github.com/hawkular/hawkular-apm-opentracing-javascript |Scheduled for 0.12
+|===
+
+Note: It is also possible to use OpenTracing compliant zipkin providers. However Hawkular APM and Zipkin providers cannot be mixed within the same environment (i.e. propagation of state is not compatible between the two projects).
+
+==== JVM based agent
+
+A java agent can be configured to non-intrusively monitor your applications/services and report trace information to the APM server.
+
+==== Zipkin compliant client libraries
+
+The http://zipkin.io/[Zipkin] project provides client libraries for instrumenting multiple languages and frameworks.
+
+Hawkular APM provides support for collecting zipkin span data via HTTP and Kafka.
+
+=== Aggregated view of components
+
+The _Components_ view provides information about traced components. The components include the _Producers_ and _Consumers_ that are involved in service invocations, as well as internal components such as _Databases_, _EJBs_, etc.
+
+A filter can be specified to focus on the time range and specific properties.
+
+ifndef::env-github[]
+image::/img/hawkular-apm/components.png[Components,400,400]
+endif::[]
+ifdef::env-github[]
+image::../../../../../assets/img/hawkular-apm/components.png[Components,400,400]
+endif::[]
+
+The chart shows the average durations of the various component types being viewed, subject to the defined filter.
+
+The table gives a more detailed breakdown of the components, identifying their average actual and elapsed durations.
+
+=== Distributed tracing
+
+The _Distributed Tracing_ tab initially presents an aggregated view of the service dependencies based on a selected initial endpoint. The nodes are colour coded to enable users to see where most of the time is being spent.
+
+As with the previous components view, it is possible to define a filter, based on time, properties and transaction, to focus in on specific information of interest.
+
+ifndef::env-github[]
+image::/img/hawkular-apm/distributed-tracing.png[Distributed Tracing - Service Dependency View,400,400]
+endif::[]
+ifdef::env-github[]
+image::../../../../../assets/img/hawkular-apm/distributed-tracing.png[Distributed Tracing - Service Dependency View,400,400]
+endif::[]
+
+At the top of the page, a button will show the number of trace instances that contribute to the aggregated service dependency view. Pressing the button results in a table being shown listing the trace instances. It is then possible to select the 'detail' icon to see a more detailed view of the trace instance.
+
+ifndef::env-github[]
+image::/img/hawkular-apm/distributed-tracing-instance.png[Distributed Tracing - Instance View,400,400]
+endif::[]
+ifdef::env-github[]
+image::../../../../../assets/img/hawkular-apm/distributed-tracing-instance.png[Distributed Tracing - Instance View,400,400]
+endif::[]
+
+
+=== Business (or user defined) transactions
```

Comment:
This parenthesis looks bit weird in heading. What could be used instead? `=== Transactions` or just `Business transactions` as it is mentioned in description.